apps/commudle-admin/src/app/app-shared-components/hackathon-horizontal-card/hackathon-horizontal-card.component.html

```diff
@@ -0,0 +1,83 @@
+<nb-card>
+  <nb-card-body>
+    <div class="upcoming-card">
+      <a
+        [routerLink]="['/communities', hackathon.kommunity_slug, 'hackathons', hackathon.slug]"
```

Comment:
For future there will be one more case , hackathon in under another parent, like hackathon will be under  communityGroup